setup.py

```diff
@@ -34,7 +34,6 @@
     license=about['__license__'],
     packages=['edgar'],
     keywords=['sec', 'edgar', 'financials', 'stock', 'fundamental', 'analysis'],
-    python_requires="==3.7",
```

Comment:
I’m not clear on why the python version requirement would be removed. I know that i had to upgrade my version of python for certain parts of the code (can’t specifically remember which) and it’s also generally best practice to establish a baseline version, so perhaps we can do a “>= 3.7“, or whatever the syntax is.

Reply:
fixed